client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -49,9 +50,21 @@ class EditorRevisionsList extends PureComponent {
 	}
 
 	render() {
+		// NOTE: This supports revisions that have been hydrated with author
+		// info (`author` is an object) and the ones that haven't (author is a
+		// string, containing just the ID ).
+		const usersId = uniq( map( this.props.revisions, r => get( r, 'author.ID', r.author ) ) );
```

Comment:
We avoid single letter var names for clarity. Should be `revision` here. It's also a bit weird that we are assigning this on a render method. Wouldn't a selector handling this make more sense?